AutoMatrix/AutoMatrix.py

```diff
@@ -468,10 +470,14 @@ def ProcessVolume(self)-> None:
                         print("not a .nii.gz")
 
                     self.UpdateTime()
-                    if extension_scan!=".nii.gz" and extension_scan!=".nrrd":
+                    if volume_storage.GetSupportedFileExtension(scan):
                         model = slicer.util.loadModel(scan)
```

Comment:
`slicer.util.loadVolume`

Reply:
I changed the code, when I'm testing it it's working :)

Reply:
This was just referring to a typo that `loadModel` was used here instead of `loadVolume`. That's resolved in the latest changes.